api/BaseAudioContext.json

```diff
@@ -95,7 +95,7 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "9.0"
```

Comment:
Stuff like this is scary and shows our info was bad until recently!!!

Reply:
(Not an error just yikes)

This maybe someone else updated our browser as 'true' since they weren't sure when it was added